tests/src/test/java/alluxio/server/meta/MetaDailyBackupIntegrationTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.server.meta;
+
+import alluxio.Configuration;
+import alluxio.PropertyKey;
+import alluxio.master.LocalAlluxioCluster;
+import alluxio.master.meta.MetaDailyBackup;
+import alluxio.testutils.BaseIntegrationTest;
+import alluxio.util.FormatUtils;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.time.Clock;
+import java.time.LocalDateTime;
+import java.util.regex.Matcher;
+
+/**
+ * Tests the daily primary master metadata backup.
+ */
+public class MetaDailyBackupIntegrationTest extends BaseIntegrationTest {
+  private File mBackupDir;
+  private LocalAlluxioCluster mCluster;
+
+  @Before
+  public void before() throws Exception {
+    LocalDateTime backUpTime = LocalDateTime.now(Clock.systemUTC()).plusMinutes(2);
```

Comment:
can we make this test run faster?

Reply:
I changed to 1 min, but this test may not be safe if the alluxio cluster takes too long to start and the `metaDailyBackup` schedules the next backup to be in 23H59M.